src/response/chi0.jl

```diff
@@ -283,6 +283,9 @@ end
         D = mpi_sum(D, basis.comm_kpts)  # equal to minus the total DOS
         δocc_tot = mpi_sum(sum(basis.kweights .* sum.(δocc)), basis.comm_kpts)
         δεF = δocc_tot / D
+        if !isnothing(ham.basis.model.fixed_εF)
+            δεF = zero(δεF)
+        end
```

Comment:
Put this on the `if temperature > 0`, because the whole block does nothing in case `!isnothing(ham.basis.model.fixed_εF)`.

Reply:
yeah you can remove the +